server/json-logger.cpp

```diff
@@ -206,44 +207,45 @@ void JsonLogger::set_env(vk::string_view env) noexcept {
   }
 }
 
-void JsonLogger::write_log(vk::string_view message, int type, int64_t created_at, void *const *trace, int64_t trace_size, bool uncaught) noexcept {
+
+void JsonLogger::write_log_with_demangled_backtrace(vk::string_view message, int type, int64_t created_at, void *const *trace, int64_t trace_size,
+                                                    bool uncaught) {
   if (json_log_fd_ <= 0) {
-    return;
+    return ;
   }
 
   auto *json_out_it = buffers_.begin();
   for (; json_out_it != buffers_.end() && !json_out_it->try_start_json(); ++json_out_it) {
   }
   assert(json_out_it != buffers_.end());
 
-  json_out_it->append_key("version").append_integer(release_version_);
-  json_out_it->append_key("type").append_integer(type);
-  json_out_it->append_key("created_at").append_integer(created_at);
-  json_out_it->append_key("env").append_string(env_available_ ? env_ : vk::string_view{});
+  write_general_info(json_out_it, type, created_at, uncaught);
 
-  json_out_it->append_key("tags").start<'{'>();
-  if (tags_available_) {
-    json_out_it->append_raw(tags_);
+  KphpBacktrace demangler{trace + 2, static_cast<int32_t>(trace_size - 2)};
```

Comment:
I think it's not needed here
```suggestion
  KphpBacktrace demangler{trace, static_cast<int32_t>(trace_size)};
```